cloud/src/recycler/recycler.cpp

```diff
@@ -1273,17 +1274,26 @@ int InstanceRecycler::delete_rowset_data(const doris::RowsetMetaCloudPB& rs_meta
     const auto& rowset_id = rs_meta_pb.rowset_id_v2();
     int64_t tablet_id = rs_meta_pb.tablet_id();
     // process inverted indexes
-    std::vector<int64_t> index_ids;
+    std::vector<std::pair<int64_t, std::string>> index_ids;
     index_ids.reserve(rs_meta_pb.tablet_schema().index_size());
     for (auto& i : rs_meta_pb.tablet_schema().index()) {
-        index_ids.push_back(i.index_id());
+        index_ids.push_back(std::make_pair(i.index_id(), i.index_suffix_name()));
```

Comment:
will this handle variant subcolumns's indexes?

Reply:
index_suffix_name is for variant